src/Cis194/Hw/Week1.hs

```diff
@@ -5,19 +5,25 @@ module Cis194.Hw.Week1 where
 -------------
 
 toDigits :: Integer -> [Integer]
-toDigits x = [x]
+toDigits = reverse . toDigitsRev
 
 toDigitsRev :: Integer -> [Integer]
-toDigitsRev x = [x]
+toDigitsRev x | x <= 0 = []
+toDigitsRev x = mod x 10 : toDigitsRev (div x 10)
 
 doubleEveryOther :: [Integer] -> [Integer]
-doubleEveryOther xs = xs
+doubleEveryOther = flipMap (zipWith (*) (cycle [1, 2]))
```

Comment:
👍 for a concise and elegant solution. And for knowing which tools you have in the toolbox (zipWith, cycle).